week-14/practices/12-context-practices/starter/src/index.css

```diff
@@ -0,0 +1,73 @@
+body {
+  /* background-image: url("https://wallpapercave.com/wp/n9cIpdX.jpg");
```

Comment:
Perhaps no comments

Reply:
fun fact- I was torn with a basic gray or this wild looking space background XD